src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -35,6 +35,11 @@ private IdentityModelEventSource()
         /// </summary>
         public static bool ShowPII { get; set; } = false;
 
+        /// <summary>
+        /// Flag which indicates whether or not armed security artifact is shown in logs when <see cref="ShowPII"/> is set to true. False by default.
+        /// </summary>
+        public static bool ShowArmedSecurityArtifact { get; set; } = false;
```

Comment:
What is the end to end for libraries using it?
How does it integrate with the IIdentityLogger?